circle.yml

```diff
@@ -1032,6 +1033,31 @@ jobs:
           working_directory: test-binary
           command: $(npm bin)/cypress info
 
+  test-types-cypress-and-jest:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/
+      # make sure we have cypress.zip received
+      - run: ls -l
+      - run: ls -l cypress.zip cypress.tgz
+      - run: mkdir test-cypress-and-jest
+      - run: node --version
+      - run: npm --version
+      - run:
+          name: Create new NPM package
+          working_directory: test-cypress-and-jest
+          command: npm init -y
+      - run:
+          name: Test types clash
+          working_directory: test-cypress-and-jest
```

Comment:
```suggestion
      - run: mkdir /root/test-cypress-and-jest
      - run: node --version
      - run: npm --version
      - run:
          name: Create new NPM package
          working_directory: /root/test-cypress-and-jest
          command: npm init -y
      - run:
          name: Test types clash
          working_directory: /root/test-cypress-and-jest
```

Reply:
oooohhh, good catch @sainthkh - I was thinking about that later, that I don't know precisely how TypeScript finds types, and this is important. I think I need to make sure _every_ test job that needs to run in isolation works from the root